schema.sql

```diff
@@ -3,8 +3,10 @@
 CREATE TABLE animals (
     id int generated by default as identity,
     name varchar(100) not null,
+    species varchar(100),
```

Comment:
- [ ] Even though this does the trick, the purpose here was for you to use the `ALTER TABLE` SQL function. So instead of simply adding a new column to the table creation, you can add something like 
```
ALTER TABLE animals ADD species varchar(255)
```

So this way, your database will modify the original animals table and add a new column.
Kindle fix this issue.

Reply:
Hello @Lordkaito 

Thanks for pointing out this error.

I have just fixed it based on the recommendation.